tests/functional/settings_secondary_emails.js

```diff
@@ -97,7 +97,9 @@ define([
         // send and open verification in same tab
         .then(click(selectors.EMAIL.UNLOCK_SEND_BUTTON))
         .then(openVerificationLinkInNewTab(email, 0))
+        .sleep(1000)
         .then(switchToWindow(1))
+          .sleep(1000)
           // panel becomes verified and opens add secondary panel
           .then(visibleByQSA(selectors.EMAIL.INPUT))
```

Comment:
Here's what the screen looks like when this fails - https://screencap.co.uk/images/ffcb2a6f0b4f670de3bdb1a734612c23b64a59ab.png

The page isn't loaded yet and visibleByQSA does not test to ensure the element exists before running its JS. I think this is solvable by adding a `testElementExists` either before this line, or in the `visibleByQSA` function [before line 96](https://github.com/mozilla/fxa-content-server/blob/302e84f9de59cbf6beed7fc1c8156db30a6c2c0a/tests/functional/lib/helpers.js#L96).